src/DataLayer/converters/tests/fixtures/Afm.fixtures.ts

```diff
@@ -901,3 +901,48 @@ export const reducedMultipleSorts: IFixture = {
         ]
     }
 };
+
+export const nativeSubTotals = {
+    afm: {
+        attributes: [
+            {
+                displayForm: {
+                    identifier: '1'
+                },
+                localIdentifier: 'a1'
+            },
+            {
+                displayForm: {
+                    identifier: '2'
+                },
+                localIdentifier: 'a2'
+            }
+        ],
+        measures: [
+            {
+                alias: 'Sum of Bundle cost',
+                definition: {
+                    measure: {
+                        aggregation: 'sum',
+                        item: {
+                            identifier: 'metric.id'
+                        }
+                    }
+                },
+                format: '#,##0.00',
+                localIdentifier: 'm1'
+            }
+        ],
+        nativeTotals: [
+            {
+                attributeIdentifiers: [],
+                measureIdentifier: 'm1'
+            },
+            {
+                attributeIdentifiers: ['a1'],
+                measureIdentifier: 'm1'
+            }
+        ]
+    },
+    resultSpec: {}
```

Comment:
native subtotals should be defined here as well in resultSpec

Reply:
I think this is handled somewhere else `toAfmResultSpec` uses `convertResultSpec` which only handles sorts not totals
```
function convertResultSpec(
    visObj: VisualizationObject.IVisualizationObjectContent
): AFM.IResultSpec {
    const sorts = convertSorting(visObj);
    // Workaround because we can handle only 1 sort item for now
    const sortsProp = sorts.length ? { sorts: sorts.slice(0, 1) } : {};

    return {
        ...sortsProp
    };
}
```

Reply:
It's handled in pivot table `getPivotTableDimensions`

Reply:
I think we resolved this

Reply:
you're right ;)